rootbeerlib/src/main/java/com/scottyab/rootbeer/Const.java

```diff
@@ -51,7 +53,9 @@ private Const() throws InstantiationException {
             "/system/bin/failsafe/",
             "/system/sd/xbin/",
             "/system/usr/we-need-root/",
-            "/system/xbin/"
+            "/system/xbin/",
+            "/cache/magisk"
```

Comment:
This and the line below need a trailing / otherwise
`String completePath = path + filename;` is going to mess up

Reply:
Might be better we change 
`             
String completePath = path + filename;
File f = new File(completePath);
`
to

`
File f = new File(path, filename);
`

To make it more robust to separators.

Line 175/176 of RootBeer.java

Reply:
good catch